src/autoscaler/mta.tpl.yaml

```diff
@@ -73,7 +105,7 @@ modules:
     parameters:
       memory: 1G
       disk-quota: 1G
-      instances: 0 # overidden in build-extension-file.sh
+      instances: 0
```

Comment:
bring the comment back

Reply:
i removed instances from all modules instead to simplify base mta ae13511f8